src/styles/blocks/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  box-sizing: border-box;
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card--position_bottom:not(:nth-last-child(-n + 4)) {
+  margin-bottom: 46px;
+}
+
+.card__picture {
+  margin: 0 auto;
+  padding: 16px 0 40px;
+  height: 134px;
+}
+
+.card__img {
+  height: 100%;
+  width: auto;
+  object-position: center;
+}
+
+.card__title {
+  font-weight: 500;
```

Comment:
if you have fw 500 by default, you can not set it here)

Reply:
fw for title isn't deleted because of h2 selector
@OlEzhA0